enclave-tls/samples/enclave-tls-server/server.c

```diff
@@ -264,8 +324,8 @@ int main(int argc, char **argv)
 		case 'p':
 			port = atoi(optarg);
 			break;
-		case 'P':
-			debug_enclave = false;
+		case 'D':
+			debug_enclave = true;
```

Comment:
The default value of debug_enclave should be false.

Reply:
yes, default has been modified false, but here it's set for true by -D